File: SmartVitals_App/SmartVitals_App/src/usf/smartvitals/json/Models/AuthenticationToken.java
package usf.smartvitals.json.Models;
import com.google.gson.annotations.*;

public class AuthenticationToken
{
	@SerializedName("IsAuthenticated")
    public boolean IsAuthenticated;
	
	@SerializedName("Message")
    public String Message;
	
	@SerializedName("Role")
    public String Role;

	@SerializedName("Profile")
	public UserInfo Profile;
	
	@SerializedName("SessionId")
	public String SessionId;
}

File: SmartVitals_Server/SmartVitals_Server/Models/json/AuthenticationToken.cs
using System;


namespace SmartVitals_Server.Models.json
{
    public class AuthenticationToken
    {
        public bool IsAuthenticated { get; set; }
        public string Message { get; set; }
        public string Role { get; set; }
        public UserInfo Profile { get; set; }
        public Guid SessionId { get; set; }
    }

}
